connect.js

```diff
@@ -18,6 +18,7 @@ const types = {
 };
 
 const excludedProps = ['anyTouched', 'mutator', 'connectedSource'];
+const _registeredEpics = [];
```

Comment:
@ryandberger could we use a hash here `{}` instead of the array? It will help with the performance a bit (we could skip `includes` and just do `if (_registeredEpics[key]) { ...` down below.

Reply:
Completed. Thanks for the suggestion!